### ISSUE TITLE
0.15.3 tarball does not include license, tests, docs...

### ISSUE THREAD

**Issue by bollwyvl** [crsmithdev/arrow#710]
Thanks for arrow, and congratulations on the release!

The change in #679 caused the released source distribution to not include the `LICENSE`, the tests, or the docs source.

Test:
```
curl -O https://files.pythonhosted.org/packages/ee/03/89730d72d6c13aee381508eb0516168546ed6f08a4bf8f5b24522ed23491/arrow-0.15.3.tar.gz
tar -xaf arrow-0.15.3.tar.gz
cat arrow-0.15.3/LICENSE
>>> cat: arrow-0.15.3/LICENSE: No such file or directory
```

The wheel includes the license (through separate metadata), but not the tests. Downstreams, such as [conda-forge](https://github.com/conda-forge/arrow-feedstock/pull/32) use these to ensure repackaging.

Will make a PR to put the `.in` back into `MANIFEST`!

**Comment by jadchaar** [comment]
Hi @bollwyvl, thanks for reporting this. I will get the fix merged and ship `v0.15.4` ASAP.

**Comment by bollwyvl** [comment]
Thank _you_ for the speedy review and merge!

**Comment by jadchaar** [comment]
@bollwyvl renaming was totally a mistake on my end--I read somewhere that MANIFEST could be used instead and I wanted to line up the naming with LICENSE, but I guess that is not the case.

I have published v0.15.4 with the fixed packaging. Thanks again for reporting this :D!
